e2e/test/scenarios/admin-2/whitelabel.cy.spec.js

```diff
@@ -145,7 +145,7 @@ H.describeEE("formatting > whitelabel", () => {
             cy.findByRole("searchbox", {
               name: "Login and unsubscribe pages",
             }).click();
-            H.popover().findByText("Custom").click();
+            H.selectDropdown().findByText("Custom").click();
```

Comment:
Yeah. The word 'popover' is ambiguous - it can refer to any floating item, or some subset of our floating items (such as Mantine's Popover) or a select dropdown (which might be a popover under the hood).